week_1/assignment.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func main() {
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Print(">")
```

Comment:
This seems trivial but `fmt.Print(">")` repeat several times, and enough for us to separate to a separate function, keep it DRY